tests/test_schema.py

```diff
@@ -158,9 +164,9 @@ def test_include_data_with_single(self, post):
         assert 'first_name' in author['attributes']
 
     def test_include_data_with_all_relations(self, post):
-        data = PostSchema(include_data=('author', 'post_comments')).dump(post).data
+        data = PostSchema(include_data=('author', 'post_comments', 'post_comments.author')).dump(post).data
         assert 'included' in data
-        assert len(data['included']) == 3
+        assert len(data['included']) == 5
```

Comment:
The tests are failing. I assume it's because the order of the included data isn't deterministic. 

I meant to say that you should add an assertion that the correct serialized authors are in the included data.

Something like 

```python
expected_post_author_ids = set([comment.author.id for comment in post.comments])
included_post_author_ids = set([each['id'] for each in data['included'] if each['type'] == 'people'])
assert included_post_author_ids == expected_post_author_ids
```